src/core/utils/EventNormalizer.ts

```diff
@@ -873,8 +873,8 @@ export class EventNormalizer {
         compiledEvent.type = compiledEvent.type || ev.type;
         compiledEvent.key = ev.key;
         compiledEvent.altKey = ev.altKey;
-        compiledEvent.ctrlKey = ev.ctrlKey;
-        compiledEvent.metaKey = ev.metaKey;
+        // Meta key not supported: we consider it equivalent to Ctrl key.
```

Comment:
Oh sorry, I meant the *manager* should consider it the same for the purpose of our application, not the *normalizer* which is supposed to be ignorant of what we do with its signals ! ;-) (And besides, I'd do this only on Mac as this is very specific to MacOS.)

Reply:
Problem is, when doing that, it seems the EventManager doesn't receive the right event, because somehow the normalizer has modified it. Can that be?